udata/core/dataset/rdf.py

```diff
@@ -751,6 +752,14 @@ def dataset_from_rdf(graph: Graph, dataset=None, node=None, remote_url_prefix: s
 
     dataset.title = rdf_value(d, DCT.title)
     if not dataset.title:
+        external_rdf = rdf_value(d, FOAF.isPrimaryTopicOf)
```

Comment:
I am unsure this is a valid way of identifying datasets that are just referenced.
I think updating the `missing title on dataset` with these considerations of why title is missing may be better?

Reply:
It may not be the only way but it is a way. The only problem is if there is a `isPrimaryTopicOf` but it's not why there is no title? Otherwise we may show "missing title" for referenced dataset in some case but it's not a big deal? (we can add more conditions later if we encounter them?)